### PR TITLE
Add support for kube-up to deploying Calico network-policy to GCI mas…

### PR DESCRIPTION
**Pull request by matthewdupre**
I can think of one notable thing that's wrong with this: it'll break the old debian master salt deploy.

Now, that's arguably moot, since I can't figure out how to run that anyway. Some reasonable options:
1. Make the PR look like it might still work by adding some additional debian chunks for etcd and such. I won't be able to test it though.
2. Make this PR look like it will still work, by copying all the shared files. Still won't know if debian master actually works though.
3. Just rip out debian master.
4. Leave it as is (isn't the whole thing scheduled for demolition anyway?).

### CODE REVIEW

File: cluster/gce/gci/configure-helper.sh [hunk 1]
```diff
@@ -520,10 +525,23 @@ function start-etcd-servers {
...
+
+  if [[ "${NETWORK_POLICY_PROVIDER:-}" == "calico" ]]; then
+    prepare-log-file /var/log/etcd-calico.log
+    PUBLIC_IP=$(ifconfig eth0 | grep 'inet ' | awk '{print $2}')
+    prepare-etcd-manifest "-calico" "6666" "2382" "60m" "0.0.0.0" "$PUBLIC_IP" "etcd-calico.manifest"
+  fi
+}
+
+# Starts Calico policy controller pod.
+# More specifically, it prepares dirs and files, sets the variable value
```
Comment (caseydavenport) [inline cluster/gce/gci/configure-helper.sh]: Looks like this just copies the file, not that other stuff?

File: cluster/saltbase/salt/calico/calico-policy-controller.manifest [hunk 2]
```diff
@@ -1,36 +1,36 @@
...
     kubernetes.io/cluster-service: "true"
-    k8s-app: calico-policy
 spec:
   hostNetwork: true
   containers:
+    # The Calico policy controller.
     - name: policy-controller
       image: calico/kube-policy-controller:v0.2.0
       env:
-      - name: ETCD_AUTHORITY
```
Comment (caseydavenport) [inline cluster/saltbase/salt/calico/calico-policy-controller.manifest:15]: Bring this back!

File: cluster/saltbase/salt/calico/calico-policy-controller.manifest [hunk 3]
```diff
@@ -1,21 +1,41 @@
 apiVersion: v1
 kind: Pod
 metadata:
-  name: calico-policy-controller
+  name: policy-controller
   namespace: kube-system
   labels:
+    version: "v0.2.0"
```
Comment (caseydavenport) [inline cluster/saltbase/salt/calico/calico-policy-controller.manifest]: Last time, Tim said to remove the "version" and "projectcalico.org/app" labels.

Comment (matthewdupre) [inline cluster/saltbase/salt/calico/calico-policy-controller.manifest]: Exactly what labels did he want then?

I'll do the same for this and the calico-node manifest.

**Pull request merged**